gpiocontrol/source/c/CMakeLists.txt

```diff
@@ -12,33 +12,40 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+#
 
 cmake_minimum_required(VERSION 2.8.8)
 
-project(C-SDK-sample)
-enable_language(C)
-
-# Target-independent flags.
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -Wall -Wextra -g")
+project(C-SDK-sample C)
 
 # Our application name.
-set(APP_NAME demo_client.afx)
+set(APP_NAME demo_client)
 
-# Demo executable.
-add_executable(${APP_NAME} src/kaa_demo.c)
+# Target-independent flags.
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -g -Wall -Wextra")
```

Comment:
We should stick with `-std=c99` and don't use gnu extensions as much as possible